envi/config.py

```diff
@@ -180,10 +180,16 @@ def parseConfigOption(self, optstr):
             valstr = str(int(valstr, 16))
 
         if not (valstr.startswith('"') and valstr.endswith('"')):
-            try:
-                int(valstr)
-            except:
-                valstr = '"' + valstr + '"'
+            if valstr.lower() == 'true':
```

Comment:
Nah, it's all good.  Just dont really like the "if it's not an int it's a string" thing :)  We can let it ride.